burnman/seismic.py

```diff
@@ -170,49 +201,77 @@ def depth(self, pressure):
         """
         raise ValueError, "not implemented"
         return -1
+    
+
 
     def gravity (self, depth):
         """
         Parameters
         ----------
         depth : float or array of floats
-        Depth(s) [m] to evaluate gravity at.
+            Depth(s) [m] to evaluate gravity at.
         
         Returns
         -------
         gravity : float or array of floats
-        Gravity for given depths in [m/s^2]
+            Gravity for given depths in [m/s^2]
         """
         raise ValueError, "not implemented"
         return -1
 
 
-class SeismicRadiusTable(Seismic1DModel):
+class SeismicTable(Seismic1DModel):
```

Comment:
Hmm, might be old. I think seven months ago, I tried to make it flexible
for both radius and depth, but have since taken that out again. But mostly
we search by depth anyways, so I think this shorter name is fine.

On Tue, Sep 15, 2015 at 7:58 PM, Ian Rose <notifications@github.com> wrote:

> In burnman/seismic.py
> <https://github.com/geodynamics/burnman/pull/107#discussion_r39550681>:
>
> >          """
> >          raise ValueError, "not implemented"
> >          return -1
> >
> >
> > -class SeismicRadiusTable(Seismic1DModel):
> > +class SeismicTable(Seismic1DModel):
>
> Any reason for the name change? Not that I object to it...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/burnman/pull/107/files#r39550681>.
>